src/Compilers/Core/Portable/InternalUtilities/SpanUtilities.cs

```diff
@@ -20,5 +20,8 @@ internal static class SpanUtilities
 
             return true;
         }
+
+        public static bool All<TElement>(this ReadOnlySpan<TElement> span, Func<TElement, bool> predicate)
+            => span.All(predicate, static (item, predicate) => predicate(item));
```

Comment:
>=> span.All(predicate, static (item, predicate) => predicate(item)); [](http://example.com/codeflow?start=12&length=68)

I think we should just clone the body of the method above and make necessary adjustments instead of creating delegates that call other delegates.